lib/OpenQA/Worker/Engines/isotovideo.pm

```diff
@@ -368,7 +368,7 @@ sub engine_workit {
         collected => sub {
             my $self = shift;
             eval { log_info("Isotovideo exit status: " . $self->exit_status, channels => 'autoinst'); };
-            $job->stop($self->exit_status == 0 ? 'done' : 'died');
+            $job->stop($self->exit_status == 0 ? 'done' : 'died: See log output for details');
```

Comment:
Or maybe even "isotovideo existed with non-zero return code, see logs for details"?

Reply:
I don't think it helps to know that the exit code is non zero if one doesn't know when it's supposed to be zero or non-zero. I don't even know if it's non-zero if a fatal test module failed :)

Reply:
Non-zero means "bad" and zero means "good". This rule is obviously used to decide whether we consider the test "died" or "done". So the message helps in the sense that it makes this decision more transparent.

Reply:
I mean, a single soft-failed module and probably not even a failed module will cause a "non-zero" exit code, right?

Reply:
updated. I shortened the message a bit, also to prevent a line break in source code due to tidy rules ;)